packages/debugger-extension/src/index.ts

```diff
@@ -1049,6 +1049,61 @@ const main: JupyterFrontEndPlugin<void> = {
       }
     }
 
+    app.contextMenu.addItem({
+      selector: '.jp-DebuggerKernelSource-source',
+      rank: 1,
+      command: '@jupyterlab/debugger:open-kernel-source'
```

Comment:
Across the JupyterLab codebase we keep command ids in a single place (always named `CommandIDs`) so that the id is not hardcoded in multiple places. Can you do that?